specify/specify7
# Fixes agent merging failing unexpectedly

Fixes #4685

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- use herb32124 db
- merge two agents 

- [ ] verify it works 

- merge any other agents with any other db and verify it works as expected
